nrpy/examples/nrpyelliptic_conformally_flat.py

```diff
@@ -35,7 +35,8 @@
 # Code-generation-time parameters:
 project_name = "nrpyelliptic_conformally_flat"
 grid_physical_size = 1.0e6
-t_final = grid_physical_size
+t_final = grid_physical_size  # This parameter is effectively not used in NRPyElliptic
+N_final = 10000  # Sets the maximum number of relaxation steps
```

Comment:
Don't we have a better stop condition? We should implement it!

Reply:
Implemented here https://github.com/nrpy/nrpy/pull/19/commits/a81357673887e39363dd28e9050eba26b1127f46 and here https://github.com/nrpy/nrpy/pull/19/commits/071fe26a3b609acc75731c7ba2c1040201dbac93.